src/components/ha-hls-player.ts

```diff
@@ -231,6 +277,12 @@ class HaHLSPlayer extends LitElement {
         width: 100%;
         max-height: var(--video-max-height, calc(100vh - 97px));
       }
+
+      ha-alert {
+        display: block;
+        padding-top: 50px;
+        padding-bottom: 50px;
```

Comment:
Maybe this makes it look slightly less weird?
```suggestion
        padding: 100px 16px;
```